### ISSUE TITLE
Wrong optimization flags in common.mk files

### ISSUE THREAD

**Issue by ggeorgakoudis** [utahfmr/VULFI#6]
I'm a user of the VULFI code. The CXX_FLAGS variable in the common.mk file of the examples is wrongly set to -o3 instead of -O3, meaning clang does no optimization.

Thanks,
Giorgis

**Comment by vcsharma** [comment]
Thanks for reporting, we'll fix it in the next release.

**Comment by vcsharma** [comment]
@vinutah Can you please fix the optimization flags as pointed by @ggeorgakoudis in below files:

examples/c++/src/common.mk 
examples/c/src/common.mk

**Comment by ggeorgakoudis** [comment]
No problem, I've also have a couple of more issues related to building on mac osx and using an LLVM debug build. Would you like me to open separate issues for these? I have fixes for those problems too. If you are interested I could open a pull request.

**Comment by vcsharma** [comment]
@ggeorgakoudis, we would love to hear if you may have any additional feedback and / or your experience using VULFI tool. Please drop us an email:  vulfi-dev@googlegroups.com

**Comment by vcsharma** [comment]
@ggeorgakoudis, yes, sure.  Please open a pull request and include @vinutah, @arnabd88 and myself as code reviewers. We'll try to prioritize the code review.
